doc/howto/use_cereal/index.md

```diff
@@ -0,0 +1,99 @@
+# How to use cereal {#howto_use_cereal}
+
+[TOC]
+
+This HowTo documents how to use cereal to load from an archive or store into one. Every SeqAn data structure which is
+marked as cerealisable can be used.
+
+\tutorial_head{Easy, 15 min, , }
+
+# Motivation
+
+Storing and loading already existing indices is a common use case, thanks to the cereal library doing so is incredible
+easy. This page will show you how to use cereal in SeqAn for one example. As example data structure std::vector was
+picked, but as already mentioned any SeqAn data structure that is documented as cerealisable can be used.
+
+# Storing
+
+Storing a data structure is as easy as using the `cereal::BinaryOutputArchive`.
+
+```cpp
+#include <fstream>
+#include <vector>
+
+#include <cereal/archives/binary.hpp>
+#include <cereal/types/vector.hpp>
+
+#include <seqan3/core/concept/cereal.hpp>
+#include <seqan3/std/filesystem>
+
+// Written for std::vector, other types also work.
+void store(std::vector<int16_t> const & data, std::filesystem::path tmp_file)
+{
+    std::ofstream os(tmp_file, std::ios::binary); // Where output should be stored.
+    cereal::BinaryOutputArchive archive(os);
+    archive(data);                                // Store data.
+}
+
+int main()
+{
+    std::filesystem::path tmp_file = std::filesystem::temp_directory_path()/"data.out"; // Get temp file.
+    store({1,2,3,4}, tmp_file);                                                         // Calls store on a std::vector.
```

Comment:
Although this is short, it might be a little confusing if people do not know about aggregate initialisation

```suggestion
    std::vector v{1,2,3,4};
    store(v, tmp_file);                                                         // Calls store on a std::vector.
```